EMRNotifier.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	_ "github.com/lib/pq"
+)
+
+type JobDetail struct {
+	Id          string `json:"id"`
+	JobId       string `json:"jobId"`
+	JobStatus   string `json:"jobStatus"`
+	RequestId   string `json:"requestId"`
+	Query       string `json:"query"`
+	Destination string `json:"destination"`
+}
+
+var db *sql.DB
```

Comment:
why this variable deceleration is out of init?

Reply:
So that, we can use it outside of init as well in the whole script